packages/icons/src/system-timelocation.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24">
+<path fill="#242424" d="M11.47 19.95c.92 0 1.8-.16 2.64-.42a8.62 8.62 0 0 1-1.03-1.73c-.53.75-1.1 1.08-1.61 1.08-.7 0-1.49-.62-2.15-2.02a10.54 10.54 0 0 1-.59-1.68h3.8c0-.36.07-.72.14-1.06H8.52a15.75 15.75 0 0 1 0-5.29h5.91c.14.74.2 1.54.22 2.4.32-.24.67-.42 1.04-.55a16.56 16.57 0 0 0-.18-1.86h2.9c.23.59.37 1.2.44 1.85.4.12.77.31 1.1.54a8.47 8.47 0 1 0-8.47 8.74zm0-15.9c.7 0 1.49.63 2.14 2.04.24.48.43 1.05.6 1.68H8.73c.17-.63.36-1.2.6-1.68.64-1.4 1.43-2.03 2.13-2.03Zm-3.1 1.6a11.13 11.13 0 0 0-.72 2.13H5.04a7.44 7.44 0 0 1 4-3.3c-.26.36-.49.74-.68 1.16Zm-.93 3.17a16.92 16.93 0 0 0 0 5.3h-2.9a7.4 7.4 0 0 1 0-5.29h2.88Zm.2 6.36c.2.78.43 1.5.72 2.13.2.43.43.83.69 1.18a7.44 7.44 0 0 1-4-3.3Zm6.24-10.72a7.44 7.44 0 0 1 4.01 3.3h-2.6a11.13 11.13 0 0 0-.72-2.13 7.02 7.02 0 0 0-.68-1.17ZM21 15.36c0-2.16-1.56-3.9-3.7-3.9-2.17 0-3.72 1.74-3.72 3.9 0 1.61 1.05 3.74 3.46 5.55.14.12.36.12.52 0 2.4-1.8 3.44-3.94 3.44-5.55Zm-2.11-.19a1.59 1.6 0 1 1-3.18 0 1.59 1.6 0 0 1 3.18 0z"/>
```

Comment:
I think you don't want the color in the svg. Remove `fill="#242424"`?

Reply:
updated